WinImageBuilder.psm1

```diff
@@ -874,6 +874,42 @@ function Run-Sysprep {
     Remove-VM $Name -Confirm:$false -Force
 }
 
+function Get-ImageInformation {
+    Param(
+        [Parameter(Mandatory=$true)]
+        [string]$driveLetter
+    )
+
+    if (Test-Path "$driveLetter\Windows\system32\ntdll.dll") {

```

Comment:
add a variable for: "$driveLetter\Windows\system32\ntdll.dll and use it afterwards